ubuntu/README.md

```diff
@@ -4,7 +4,7 @@ Using Ubuntu
 Add it to you systems keyboard and restart your computer.
 
 ```
-cp real-prog-dvorak >> ~/.config/xkb/real-prog-dovark
+cp real-prog-dvorak ~/.config/xkb/real-prog-dovark
```

Comment:
```suggestion
cp real-prog-dvorak ~/.config/xkb/real-prog-dvorak
```

Reply:
I believe that `/symbols` is necessary, didnt work without for me